pom.client.build.xml

```diff
@@ -1,19 +1,14 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
I assume the plan is to follow-up with another PR later to remove this file once you can update the azure pipeline to point to the new project file.

Reply:
yeah, once the migration is successfully done. The plan is to delete this file.